content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_completed.rb

```diff
@@ -55,6 +72,8 @@ def check_completed(service)
                   end
                 else
                   @handle.root['ae_result'] = 'retry'
+                  @handle.root['ae_retry_interval'] = 1.minute
```

Comment:
@billfitzgerald0120 Since we setup a method for calculating `ae_retry_interval` I would expect this default value to move into there to keep the logic all together.  It is also better for testing.